src/main/kotlin/no/nav/klage/document/api/DocumentController.kt

```diff
@@ -102,6 +102,20 @@ class DocumentController(
         )
     }
 
+    @Operation(
+        summary = "Hent dokument-json",
+        description = "Hent dokument-json"
+    )
+    @ResponseBody
+    @GetMapping("/{documentId}/json")
+    fun getDocumentAsJson(
```

Comment:
Noe særlig å tjene på denne i stedet for å bruke `getDocument`?

Reply:
Her hadde det vært lurt å se på hva som fantes i appen fra før. Du har helt rett, det er jo bare å bruke `getDocument`. Jeg skal lage en som bare har Sleuth.